MMEX/Views/Assets/AssetDetailView.swift

```diff
@@ -9,7 +9,8 @@ import SwiftUI
 
 struct AssetDetailView: View {
     @EnvironmentObject var env: EnvironmentManager // Access EnvironmentManager
-    @Binding var asset: AssetData
+    @Binding var allCurrencyName: [(Int64, String)] // Bind to the list of available currencies
+    @State var asset: AssetData
```

Comment:
can we keep @Binding here to retain a two-way connection? otherwise, asset editing can't be reflected in time in listview.

Reply:
the Preview gives an error with Binding. can you fix it?

Reply:
Refer to txnListView2<-> txnDetail , it also maintains a map of array. 
I am inclined to simplify it to an array

Reply:
i see. i need to add a complex binding. i'll do it tomorrow.